resources/crds/ripsaw_v1alpha1_hammerdb_cr.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
```

Comment:
Can we just have a single CR, and just comment out the ES parts? I know some of the workloads have index CRs, however, I think it is unnecessary to drag two CRs around that basically do the same thing.